chromadb/api/async_client.py

```diff
@@ -152,17 +154,11 @@ async def heartbeat(self) -> int:
     @override
     async def list_collections(
         self, limit: Optional[int] = None, offset: Optional[int] = None
-    ) -> Sequence[AsyncCollection]:
+    ) -> List[CollectionName]:
```

Comment:
Why did this go from `Sequence` to `List` ?

Reply:
No good reason, will revert back